src/FTPProfile.cpp

```diff
@@ -51,7 +51,6 @@ FTPProfile::FTPProfile() :
 }
 
 FTPProfile::FTPProfile(const TCHAR * name) :
-	m_parent(NULL),
 	m_port(21),

```

Comment:
why is this necessary?

Reply:
Its initialized in method body, but you are right, should be initialized before